rmw_zenoh_cpp/CMakeLists.txt

```diff
@@ -22,10 +22,6 @@ find_package(rosidl_typesupport_fastrtps_c REQUIRED)
 find_package(rosidl_typesupport_fastrtps_cpp REQUIRED)
 find_package(rmw REQUIRED)
 find_package(zenoh_c_vendor REQUIRED)
-find_package(zenohc_debug QUIET)
```

Comment:
Could you clarify that the original changes are no longer needed to build in Debug mode? These lines were added in https://github.com/ros2/rmw_zenoh/pull/166

Reply:
This was on purpose but no longer needed anymore. Now we use different folders to distinguish them.